WallsAndHoles/tile.h

```diff
@@ -26,6 +26,16 @@ class Tile : public QObject
     float height() const;
     QVector2D position() const;
 
+
+    /**
+     * @brief Returns the TileMaterial if there is a TileTemplate, and otherwise
+     * returns nullptr.
+     */
+    TileMaterial *materialPtr();
```

Comment:
Probably change this to just `TileMaterial *marerial();` as that is the standard used elsewhere